src/commands/attest/create.ts

```diff
@@ -13,6 +13,12 @@ const MAX_SEARCH_RESULTS = 5
 const prompt = promptSync()
 
 export default class CreateAttestations extends Command {
+    static usage = 'attest create -o <attestation-file> [-i <hdf-json> -t <json | xlsx | yml | yaml>]'
```

Comment:
If someone wants to use `yml` or `xlsx` formats only available for suggesting control inputs, or can someone use a `xlsx` and `yml` file and skip the `attest create` step altogether?

Reply:
I'm not sure what your question is here @em-c-rod.  The format type option there is to specify the output file's type (i.e. the type for `attestation-file`) which by default is json but could be xlsx or yaml instead.  It is not necessary to use the attest create subcommand if the user creates the attestation by hand in the proper format.

Reply:
Perfect, thanks.